src/status_im/contexts/communities/home/view.cljs

```diff
@@ -43,6 +43,52 @@
    {:id :pending :label (i18n/label :t/pending) :accessibility-label :pending-tab}
    {:id :opened :label (i18n/label :t/opened) :accessibility-label :opened-tab}])
 
+(defn- community-creation-options-testing
+  []
+  [rn/view
+   {:padding-vertical 12
+    :row-gap          12}
```

Comment:
`{:style`?

Reply:
Thank you! I forgot it :man_facepalming: